src/Expander.hs

```diff
@@ -283,8 +285,8 @@ evalMod (Expanded em _) = snd <$> runWriterT (traverse_ evalDecl (view moduleBod
             modifyState $
               over (expanderWorld . worldTransformerEnvironments . at p . non Env.empty) $
                 Env.insert n x v
-        Meta decl -> do
-          ((), out) <- lift $ inEarlierPhase (runWriterT $ evalDecl decl)
+        Meta decls -> do
+          ((), out) <- lift $ inEarlierPhase (runWriterT $ traverse_ evalDecl decls)
```

Comment:
One design decision is whether the `meta` builtin should wrap a single declaration (which could be a `group`) or allow a list of declarations. I implemented the list approach.

Reply:
Seems fine to me - each can macro-express the other.